### PR TITLE
Hotkey fixes

### PR DESCRIPTION
**Pull request by skidau**
Stability fixes for the pad configurable hotkeys.

Windows test build: http://dl.dolphin-emu.org/prs/pr-2010-dolphin-latest-x64.7z

### CODE REVIEW

File: Source/Core/Core/HW/GCKeyboard.cpp [hunk 1]
```diff
@@ -35,8 +35,9 @@ void Shutdown()
 // if plugin isn't initialized, init and load config
 void Initialize(void* const hwnd)
 {
-	for (unsigned int i=0; i<4; ++i)
-		s_config.controllers.push_back(new GCKeyboard(i));
+	if (s_config.controllers.size() == 0)
```
Comment (lioncash) [inline Source/Core/Core/HW/GCKeyboard.cpp]: `if (s_config.controllers.empty())`

Comment (skidau) [inline Source/Core/Core/HW/GCKeyboard.cpp]: Have updated the code to use empty()

**Comment by ROMaster2** [comment]
It opens normally now, sweet.

Are Hotkey Settings and Menu Accelerators under Options supposed to be the same thing?

**Comment by skidau** [comment]
Yeh, the menu accelerators are the key shortcuts that get added to the menu.  Either one or both can be used and they need not be mapped to the same key.

**Pull request merged**